src/main/scala/rocket/RocketCore.scala

```diff
@@ -25,6 +25,11 @@ case class RocketCoreParams(
   useCompressed: Boolean = true,
   useRVE: Boolean = false,
   useSCIE: Boolean = false,
+  useBitManip: Boolean = false,
+  useBitManipCrypto: Boolean = false,
+  useABLU: Boolean = false,
```

Comment:
Why would user want `useBitManip && !useABLU`. Can we make `useABLU` derived?

Reply:
Actually is this still used when there is using ABLU

Reply:
Removed `useABLU`.